src/GraphQL.AotCompilationSample/GraphQL.AotCompilationSample.csproj

```diff
@@ -0,0 +1,22 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <OutputType>Exe</OutputType>
+    <TargetFramework>net7.0</TargetFramework>
+    <ImplicitUsings>enable</ImplicitUsings>
+    <Nullable>enable</Nullable>
+    <PublishAot>true</PublishAot>
```

Comment:
Also I feel some confusion about a bunch of different project options regarding trimming - [`PublishTrimmed`](https://learn.microsoft.com/en-us/dotnet/core/deploying/trimming/trim-self-contained?source=recommendations#enable-trimming), `IsTrimmable`, `PublishAOT`, etc. I think `PublishAot` is the most recent one.